### DISCUSSION TITLE
Is email working for self-hosting?

### DISCUSSION THREAD

**Discussion by kohei306** [plausible/analytics#1471] [Self-Hosted Support] [answered]
I have been trying to make the email work with thrid party smtp server but I cannot make it work..
Is it working for anyone?

I am setting those config options
MAILER_ADAPTER=Bamboo.SMTPAdapter
MAILER_EMAIL=plausible@ mymail.co.uk
SMTP_HOST_ADDR=box.mymail.co.uk
SMTP_HOST_PORT=465
SMTP_USER_NAME=plausible@ mymail.co.uk
SMTP_USER_PWD=password
SMTP_HOST_SSL_ENABLED=true
SMTP_RETRIES=5
ENABLE_EMAIL_VERIFICATION=false

but I keep getting 500 error and errors below from containers

10:35:07.492 request_id=FrhO-7J8F-jw-gUAAAzR [debug] Sending email with Bamboo.SMTPAdapter:

 Failed to send Sentry event.
plausible_1            | 10:35:07.498 request_id=FrhO-7J8F-jw-gUAAAzR [warn] Description: 'Authenticity is not established by certificate path validation'
plausible_1            |      Reason: 'Option {verify, verify_peer} and cacertfile/cacerts is missing

**Comment by kohei306** [comment, marked as answer]
it was some typo..

**Reply by metmarkosaric** [reply]
thanks for the update @rjhcnf! enjoy Plausible!

**Reply by rdvo** [reply]
how do I integrate email , when I add a user to a site it isn't sending it out. where do i set those settings? I used docker

**Reply by ruslandoga** [reply]
@ovived 👋

My guess is port 25 is blocked on your hosting provider.

Do you have access to logs? The command to get them would be something like `cd hosting && docker compose logs` for the default installation.

**Comment by bigbeka** [comment]
I am new to docker and all around it, how does one add these configuration options to a docker-compose.yml file? Or should it be added to .env file? if it is docker compose file, at which level should it be formatted and to which service? It would be helpful if someone could show an example where these options are being used.

Many thanks.
